lib/hutil.ml

```diff
@@ -150,6 +116,57 @@ let error_cannot_access conf fname =
       ^<^ (Util.escape_html fname : Adef.escaped_string :> Adef.safe_string)
       ^>^ ".txt\".")
 
+let header_with_title ?(error = false) ?(fluid = false) ?(no_title = false) conf
```

Comment:
`header_with_title` having a `no_title` argument is confusing. I would call this function `header_with_page_title` instead and maiybe somehting like `no_content_title` instead of `no_title`